cfme/fixtures/cli.py

```diff
@@ -268,8 +267,7 @@ def ha_multiple_preupdate_appliances(appliance, old_version):
         yield apps
 
 
-@pytest.fixture
-def ha_appliances_with_providers(ha_multiple_preupdate_appliances, app_creds):
+def configure_appliances_ha(appliances, pwd):
```

Comment:
All of these methods are great, but I don't think this is the right place for them.

Though we will want to use the actions contained in these methods as test setup through fixtures, the methods should be defined in a more consistent and consumable place.

None of these locations specifically jump out to me as _the_ obvious place to locate these interactions, but there are a few that come to mind.

1. Within cfme.utils.appliance.ApplianceConsole class - though it should be pulled out of __init__.py and into a plugin module.

2. As a `ViaSSH` sentaku contextual module, as some of these actions can be completed ViaUI (replication setup), and the ApplianceConsole way of doing it is just another context.

Reply:
@mshriver the replication is different thing than the one that can be set in the UI. We have a name clash between the CLI and WebUI.

Reply:
Discussed offline as well, going to move all of this in separate PR, creating an ApplianceConsole module building off of AppliancePlugin